label_sync/main.go

```diff
@@ -467,9 +467,9 @@ func rename(repo string, previous, wanted Label) Update {
 }
 
 // Update the label color/description
-func change(repo string, label Label) Update {
-	logrus.WithField("repo", repo).WithField("label", label.Name).WithField("color", label.Color).Info("change")
-	return Update{Why: "change", Current: &label, Wanted: &label, repo: repo}
+func change(repo string, previous, wanted Label) Update {
```

Comment:
Passing the previous label just for logging is not necessary. We can output a log before changing the label based on the operation we are doing. (color, description, name)

Reply:
It's not for logging, the function actually stores the same version of the label both as `Current` and `Wanted`, which is not correct in this case. If a label has been marked as "to change" it means that the color or the description (or both) differs so it would be useful the store the current version in `Current` and the to-be version in `Wanted`.

Reply:
This leads to two questions. Where this is being used, and why the tool is not broken.

Reply:
> Where this is being used  

Nowhere other than the unit tests.
> why the tool is not broken
  
See the answer above 😄 

It's something I noticed while I was debugging the tool and I think it makes sense: when I want to `change` a label (again, one of [these conditions](https://github.com/kubernetes/test-infra/blob/71922e5c3c42c84bf7646aacaca44623d24a8491/label_sync/main.go#L586-L589) is true) then `Current` holds the actual version and `Wanted` the desired one.

Reply:
Since it is not being used anywhere, it looks like we have it only for logging....